docs/guides/configuration.md

```diff
@@ -260,6 +260,67 @@ If you created a `dev` environment for a project containing a model named `my_sc
 
 The default behavior of appending the suffix to schemas is recommended because it leaves production with a single clean interface for accessing the views. However, if you are deploying SQLMesh in an environment with tight restrictions on schema creation then this can be a useful way of reducing the number of schemas SQLMesh uses.
 
+#### View catalogs
```

Comment:
Should this concept be called `Environment catalogs` instead? I feel like "view" is a broader term that extends beyond what's captured by this feature.

Reply:
I see your point but I am following the pattern from the previous titles. It goes like this:

* Physical table schemas
...
* View schemas
...
* View catalogs
....

So that matches the pattern.

Reply:
What do you think about `Environment view schemas` and `Environment view catalogs`?

Reply:
Yep, your suggestion sounds good, thanks